pkgs/development/python-modules/aiorwlock/default.nix

```
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, pytest-asyncio
+, pytest-cov
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "aiorwlock";
+  version = "1.3.0";
+
+  checkInputs = [
+    pytestCheckHook
+    pytest-asyncio
+    pytest-cov
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-g/Eth99LlyiguP2hdWWFqw1lKxB7q1nGCE4bGtaSq0U=";
+  };
```

```suggestion
  src = fetchPypi {
    inherit pname version;
    sha256 = "sha256-g/Eth99LlyiguP2hdWWFqw1lKxB7q1nGCE4bGtaSq0U=";
  };

  checkInputs = [
    pytestCheckHook
    pytest-asyncio
  ];
```
we don't want to slow down things by generating coverage we never use.

---

removed